packages/invoices/src/InboundShipment/DetailView/Footer.tsx

```diff
@@ -26,43 +27,37 @@ interface InboundDetailFooterProps {
   save: () => Promise<void>;
 }
 
-const createStatusLog = (status: 'DRAFT' | 'CONFIRMED' | 'FINALISED') => {
-  // const {
-  //   entryDatetime,
-  //   allocatedDatetime,
-  //   shippedDatetime,
-  //   pickedDatetime,
-  //   deliveredDatetime,
-  // } = draft;
+const createStatusLog = (draft: InboundShipment) => {
```

Comment:
Hmm so i'm not fully sure this is the "best" way to be doing this, but have just updated for now to handle the new statuses